src/Database/Relations/BelongsToMany.php

```diff
@@ -202,6 +200,8 @@ public function detach($ids = null, $touch = true)
          *
          */
         $this->parent->fireEvent('model.relation.afterDetach', [$this->relationName, $attachedIdList]);
```

Comment:
Should the after event params include the result of the method?

Reply:
Yeah, was asking myself the same question afterwards... @Rike-cz?

Reply:
Agree.